3rdparty/mbedtls/config.h

```diff
@@ -33,6 +33,14 @@
 #define _CRT_SECURE_NO_DEPRECATE 1
 #endif
 
+#cmakedefine MBEDTLS_OE_INCLUDE_ENCLAVE_H
```

Comment:
Would #cmakedefine cause problems with other preprocessors?  We should make sure this is as tool-agnostic as is practical.

Reply:
`#cmakedefine` is already used by Open Enclave during CMake's configuration stage.